model/code/registry.yaml

```diff
@@ -11,6 +11,7 @@ groups:
         brief: >
           The method or function fully-qualified name without arguments. The value should fit the natural representation of the language
           runtime, which is also likely the same used within `code.stacktrace` attribute value.
+          When using OTLP Profiles this attribute is covered my 'message Function' and MUST NOT be used. This constraint is imposed to prevent redundancy and maintain data integrity.
```

Comment:
(and same below)
```suggestion
          When using OTLP Profiles this attribute is covered by 'message Function' and MUST NOT be used. This constraint is imposed to prevent redundancy and maintain data integrity.
```